docs/evaluate/how-it-works.mdx

```diff
@@ -0,0 +1,90 @@
+---
+id: how-it-works
+title: How Temporal Works
+sidebar_label: How It Works
+description: This guide provides a short overview of How Temporal Works.
```

Comment:
Why random capitalization out of curiosity? How about: "This guide provides a short overview of how Temporal works."?